woocommerce/wc-admin
# Add refund filtering to order reports

Fixes #2003

Allows filtering to view different types of refunds or filter out refunds.

One odd side effect of doing this is that filtering by "None" for refunds still shows orders with a status of "Refunded" but of course won't show their respective refunds.  I believe this is expected behavior but seems odd at first glance.
<img width="347" alt="Screen Shot 2019-05-07 at 7 41 25 PM" src="https://user-images.githubusercontent.com/10561050/57296674-210dc500-7100-11e9-82fb-8df6af982441.png">
<img width="574" alt="Screen Shot 2019-05-07 at 7 41 20 PM" src="https://user-images.githubusercontent.com/10561050/57296675-210dc500-7100-11e9-87f2-9bcf9a6d0076.png">

### Screenshots
<img width="404" alt="Screen Shot 2019-05-07 at 7 37 46 PM" src="https://user-images.githubusercontent.com/10561050/57296446-9c22ab80-70ff-11e9-8712-38844b68246d.png">

### Detailed test instructions:

1. Make an order without refunds, one with a partial refund, and one with a full refund (set status to `refunded`).
2. Open the orders report.
3. Add the Refunds filter under the Advanced Filters.
4. Filter by various types of refunds and make sure stats are correct.
